docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -183,49 +182,34 @@ Among MM and RRMM patients, they expect to have:
-trends out from 2021 to 2025. For the US general population, registry population, 
-and key sub-populations, Isatuximab will start to be available to simulants as a 
-second and later lines of regimen and ramp up to a coverage rate differ by line 
-of treatment in 2025 to simulate a business-as-usual treatment scenario.
-
-**Alternative** Most aspects of the alternative scenario will be the same as the 
-baseline scenario: it will project GBD 2019 demographic and disease trends out 
-from 2021 to 2025 and apply the same coverage rates of Isatuximab regimen (as 2+ 
-line treatment) specified in the baseline. In contrast to the baseline scenario, 
-Isatuximab in the alternative scenario will start to be available to simulants 
-as a frontline regimen and ramp up to 10% coverage at frontline in 2025 among 
-all simulated population.
-
-
-In the absence of data from Flatiron, we made following assumptions:
- 1. The initial treatment coverage of Isatuximab is informed by Sanofi's 
-    commercial team.
- 2. The probability of simulants treated with Isatuximab is differ by lines of 
-    treatment and capped to not exceed 10% in 2025.
- 3. The coverage scale-up of Daratummumab follows the trend from IQVIA sales 
-    projection.
+To measure the impact of an isatuximab scale-up in the first line of treatment, we will simulate two scenarios, a baseline scenario and an alternative scenario, outlined below. The underlying health state of each simulant will be measured at each 28-day time step and the probability that each simulant is treated will be dependent on the coverage stated in that scenario. 
 
-.. note::
+**Baseline** Coverage of isatuximab in the first line of treatment will remain at its 2021 value of 0% for the duration of the simulation from 2021 to 2025.
+
+**Alternative** Coverage of isatuximab in the first line of treatment will scale-up linearly over the duration of the simulation from 0% in 2021 to 10% in 2025. 
+
+More details on treatment coverage in each simulation scenario can be found in the `5.3.3.2 Treatment Modeling Strategy`_ section.
+
+Additionally, we will perform several runs of the simulation baseline and alternative scenarios under different key parameter assumptions in order to perform sensitivity analyses that test the impact of different parameter values on simulation outcomes (the difference between scenarios). The two parameters and their respective assumed values we will include in this sensitivity analysis are listed below.
+
+1. Treatment effects (see the `5.3.3.2 Treatment Modeling Strategy`_ section for more detail)
+
+  1a. Flatiron Health based estimates
+
+  1b. Clinical trial based estimates
+
+2. Impact of race on survival outcomes (see the `5.3.2.2 Risk Factor Effects`_ section for more detail)
+
+  2a. Hazard ratios from the CoMMpass registry [Derman-et-al-2020]_
+
+  2b. Assumption of no impact of race on multiple myeloma survival outcomes
```

Comment:
Is there some evidence out that that Black race has a protective effect on survival outcomes? If so, that would be an even more extreme sensitivity analysis that we could do instead of the no-impact version.

Reply:
Yes, there is some evidence of this, although it varies by study and does not always persist after adjustment for age.

Notably, and I can clarify this in the documentation, this assumption we are making is that there is no impact of race on MM survival outcomes *after adjustment for age.* So at the aggregate level in our simulation using this hazard ratio, black race will be protective for MM survival outcomes. 

I think this is the most appropriate value to run without doing a more systematic data extraction of the age-adjusted race HR estimates (there is also some evidence that it varies by age), but I can do that too!

Reply:
@aflaxman, any updated thoughts based on this info?

Reply:
Let's not make unnecessary work, and go with the alternative hypothesis that there is no impact of race on MM survival after adjusting for age.  Can you share an explanation of this with Jonathan, so he can confirm it is acceptable?